.gitignore

```diff
@@ -8,9 +8,14 @@
 /bin
 bin/
 build/
+.DS_Store
```

Comment:
Do we have a way to ignore all DS_Store filed in children folders? I guess this file might exist there.